controllers/Frontend.php

```diff
@@ -10,52 +10,70 @@ public function home_cv()
     {
         $model_form = new Form();
         
+        $formulaire = $model_form->form_page_home_cv();
         $page="frontend/home_cv";
-        $topbar="frontend/topbar";
+        $topbar="topbar_frontend";
         require "views/layout.php";
+
+        //$myView = new View('home_cv');
+       // $myView->render;
     }
 
     public function legalnotice()
     {
         $page="frontend/legalnotice";
-        $topbar="frontend/topbar";
+        $topbar="topbar_frontend";
         require "views/layout.php";
     }
     
     public function home()
     {
         $model_article = new Article();
         $model_comment = new Comment();
+        $posts = $model_article->get_posts_blog();
+        
 
         $page="frontend/home";
-        $topbar="frontend/topbar";
+        $topbar="topbar_frontend";
         require "views/layout.php";
     }
 
     public function blog()
     {
         $model_article = new Article();
         $model_comment = new Comment();
+        $posts = $model_article->get_posts_blog_all();
         
         $page="frontend/blog";
-        $topbar="frontend/topbar";
+        $topbar="topbar_frontend";
         require "views/layout.php";
     }
 
     public function post()
     {
+        
         $model_article = new Article();
         $model_comment = new Comment();
                 
+        $responses = $model_comment->get_comments_blog();
+        $post = $model_article->get_article_blog();
+        $model_comment->form_comment_verification();
+        
+        if($post == false){
+            header("Location:/page-erreur");
+        }else{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method post uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)